shell/shared/grain.js

```diff
@@ -1358,6 +1432,19 @@ if (Meteor.isClient) {
             return "remove";
           }
         });
+      } else if (event.data.inlinePowerbox) {
+        var inlinePowerbox = event.data.inlinePowerbox;
+        check(inlinePowerbox, Object);
+        var rpcId = inlinePowerbox.rpcId;
```

Comment:
It would be good to check the type of `rpcId` here. Have we actually settled what type it is allowed to be? Your test app appears to use `Number` for this method. However, the `renderTemplate` method, above, requires that `rpcId` be a string.